### ISSUE TITLE
Travis CI installation error: `typings: not found`

### ISSUE THREAD

**Issue by sanex3339** [pleerock/class-validator#30]
I got error during Travis CI building.

```
> class-validator@0.5.0-alpha.3 postinstall /home/travis/build/sanex3339/javascript-obfuscator/node_modules/class-validator
> typings install
sh: 1: typings: not found
```

This error comes after i removed `typings` package from `package.json` of my package.

Looks like `"typings": "^1.3.1"` must be in `dependencies` instead of `devDependencies`.

**Comment by pleerock** [comment]
nope it should be in devDependency. Does npm runs postinstall scripts of class-validator when install your package?

**Comment by sanex3339** [comment]
travis log
https://travis-ci.org/sanex3339/javascript-obfuscator/jobs/147376155

**Comment by pleerock** [comment]
I think I should remove `typings install` from `postInstall` from package.json

**Comment by sanex3339** [comment]
I try to make a PR with completely remove of typings package in favor of ts 2.0.0 and @types

**Comment by pleerock** [comment]
I think for now its better to simply remove postInstall from package.json. I will solve your problem. I don't want to adopt ts 2.0 until it completely released and get a bit of people adoption.

**Comment by sanex3339** [comment]
any ETA?

**Comment by pleerock** [comment]
should be fixed in 0.5.0
